api_client/python/timesketch_api_client/sketch.py

```diff
@@ -1694,6 +1695,8 @@ def generate_timeline_from_es_index(
             name: string with the name of the timeline.
             index_name: optional string for the SearchIndex name, defaults
                 to the same as the es_index_name.
+            timeline_filter_id: optional integer to filter on documents in an
```

Comment:
Sorry my bad this should be a string

```suggestion
            timeline_filter_id: optional string to filter on documents in an
```